main.py

```diff
@@ -129,30 +129,27 @@ def show_settings(self):
             self.dialog.findChild(QtCore.QObject, RECORD_SETTINGS).setChecked(True)
         if cfg[CONFIG_MUTE]:
             self.dialog.findChild(QtCore.QObject, MUTE_SETTINGS).setChecked(True)
-        index = self.dialog.findChild(QtCore.QObject, QUALITY_SETTINGS).findText(cfg[CONFIG_QUALITY])
-        self.dialog.findChild(QtCore.QObject, QUALITY_SETTINGS).setCurrentIndex(index)
+        self.dialog.findChild(QtCore.QObject, QUALITY_SETTINGS).setText(", ".join(cfg[CONFIG_QUALITY]))
```

Comment:
Maybe extract this to its own variable for clarity? Might not be entirely clear right now what this does. Talking about the string joining.